google/cloud/bigquery/v2/minimal/benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,109 @@
+# ~~~
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ~~~
+
+function (experimental_bigquery_rest_client_define_benchmarks)
```

Comment:
These days we would probably write:

```CMake
if (NOT BUILD_TESTING)
    return ()
endif ()
```

And not use a function to scope all the variables.